shadan-abdulbasit/style.css

```diff
@@ -0,0 +1,139 @@
+body{
+  margin:0px;
+  padding:0px;
+  font-family:"Helvetica";
+}
+.container{
+  display: flex;
+ 
+} 
+
+aside{
+  
+  flex: 2;
+  background: #3d4f5d;
+  display:flex;
+  align-items:flex-start;
+  text-align: right;
+  color: white;
+  padding-left: 5px;
+  margin-right: 10px;
+  padding: 200px 0;
+  
+}
+
+aside div {
+  padding-left: 30px;
+  position:fixed; 
+
+  
+}
+
+aside div p a{
+  border: 1px solid white;
+  padding: 3px 10px;
+  text-decoration:none;
+  color:white;
+  /* font-weight:bold; */
+  margin-right:2px;
+  margin: 0 4px;
+}
+
+main{
+  flex:7;
+  padding: 2.5em;
+}
+#blog{
+  margin-bottom:-12px;
+  font-size:28px;
+  /* font-size: 24px; */
+}
+
+#pined-post , #resent-post{
```

Comment:
you can here create a class with name for example *margin-bottom* and then give these to elements this class